annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -379,7 +379,10 @@ private String resolveHgvspShort() {
                     }
                 }
             }
-            else if (canonicalTranscript.getHgvsc() != null && spliceSiteVariants.contains(canonicalTranscript.getConsequenceTerms().get(0))) {
+            else if (canonicalTranscript.getHgvsc() != null && (spliceSiteVariants.contains(canonicalTranscript.getConsequenceTerms().get(0)) || 
```

Comment:
It is safe to do this if `canonicalTranscript.getHgvsc() != null` and nothing else.